ops/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  app:
+    image: pentacent/keila
+    ports:
+      - "4000:4000"
+    command:
```

Comment:
I suggest removing this and instead adding `CMD ["start"]` to the Dockerfile.

Reply:
Do you suggest adding/publishing a `dev` Dockerfile, or changing the official image?

Reply:
I would suggest adding this in the actual `Dockerfile`; I don’t think a dockerized dev environment is something we need right now :smiley:

Reply:
I’ve already added `CMD` to the Dockerfile now :partying_face: